src/browser/convert_options.js

```diff
@@ -269,6 +269,10 @@ module.exports = {
             newOptions.preload = options.preload || [];
         }
 
+        if (newOptions.transparent) {
+            newOptions.backgroundColor = '#0FFF';
```

Comment:
would `'transparent'` work here?

if so I wonder if we really even need `options.transparent'`⁉